collect_app/src/main/java/org/odk/collect/android/utilities/AuditEventLogger.java

```diff
@@ -132,6 +137,14 @@ private void addLocationCoordinatesToAuditEventIfNeeded(AuditEvent auditEvent) {
         }
     }
 
+    private void addNewValueToAuditEventIfNeeded(AuditEvent aev) {
```

Comment:
I would tend to pull the conditional out of the method and call this `addNewValueToAuditEvent` but you could also consider `addNewValueToQuestionAuditEvent` or something like that. I know there are a number of methods with the "IfNeeded" suffix but it just doesn't convey much about what the method does.

Reply:
Done.